composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "reload/github-security-jira",
+    "description": "Create Jira tickets for GitHub security alerts",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.2.0",
+        "symfony/console": "^4",
+        "lesstif/php-jira-rest-client": "^1",
+        "symfony/yaml": "^5.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "GitHubSecurityJira\\": "src/"
+        }
+    },
+    "bin": [
+        "ghsec-jira"
```

Comment:
This should be `bin/ghsec-jira`. But it only matters when this project is installed as a dependency of another, in which case composer links the files into `vendor/bin`.